dev/dune-workspace.all

```diff
@@ -2,7 +2,7 @@
 
 ; Add custom flags here. Default developer profile is `dev`
 (env
- (dev     (flags :standard -rectypes -w -9-27-50))
+ (dev     (flags :standard -rectypes -w -9-27-50+60))
```

Comment:
Is there any way to avoid this problem by "including" the default profile?

Reply:
A future refactoring should take care of it